record-and-playback/presentation/scripts/publish/presentation.rb

```diff
@@ -355,6 +355,11 @@ def svg_render_shape_text(g, slide, shape)
 end
 
 def svg_render_shape_poll(g, slide, shape)
+  result = shape[:result]
+  if result == "[]"
```

Comment:
Hmm. I don't like doing a string comparison here, since the contents are actually JSON, so there's no guarantee that the formatting will be fixed.

That said, it is unlikely to change, and we don't actually do anything with the parsed JSON here (it's all done in the python script), so adding an extra JSON parse might be overkill.

Probably good enough as-is. Worst case is that some future recordings might not hit this check and we'll have to add a more complete fix later.